templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -120,6 +120,9 @@ contents:
           containerPort: 2379
           protocol: TCP
       hostNetwork: true
+      priorityClassName: system-node-critical
```

Comment:
@sjenning are there any other magic incantations missing from etcd here?

Reply:
from https://docs.okd.io/latest/admin_guide/scheduling/priority_preemption.html#admin-guide-priority-preemption-priority-class
```
System-cluster-critical - This priority class has a value of 2000000000 (two billion) and is used with pods that are important for the cluster. Pods with this priority class can be evicted from a node in certain circumstances. For example, pods configured with the system-node-critical priority class can take priority. However, this priority class does ensure guaranteed scheduling. Examples of pods that can have this priority class are fluentd, add-on components like descheduler, and so forth.
```

seems like cluster-critical can be evicted...